src/constants/labels.json

```diff
@@ -0,0 +1,73 @@
+{
+  "overview": {
```

Comment:
I think `main`, `home` or `general` would be more fitting. It also kind of conflicts with the exercise `overview`.

Reply:
done